packages/features/form-builder/FormBuilder.tsx

```diff
@@ -283,6 +283,14 @@ export const FormBuilder = function FormBuilder({
           }
           handleSubmit={(data: Parameters<SubmitHandler<RhfFormField>>[0]) => {
             const type = data.type || "text";
+            if ("minLength" in data || "maxLength" in data) {
+              if (Number.isNaN(data.minLength)) {
+                data.minLength = undefined;
+              }
+              if (Number.isNaN(data.maxLength)) {
+                data.maxLength = undefined;
+              }
+            }
```

Comment:
It seems hackish. What are we trying to achieve here? Even if this code is required, we should keep it abstracted out from `handleSubmit`.

Also, I think we should have hard limit on max value. Max could be as high as 4000 maybe(could be easily changed later). So, user can't set max characters beyond that MAX value.